### ISSUE TITLE
New Deploy dependency warnings

### ISSUE THREAD

**Issue by resistorsoftware** [Shopify/shopify_app#1184]
I know this is NOT specific to Shopify App, but I am fishing for an answer anyway. When I deploy a fresh rails App for a new Shopify App, I am seeing these messages constantly and I would like to remove them. What is the easiest way to do that? 

    warning @rails/webpacker > node-sass > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
    warning @rails/webpacker > node-sass > node-gyp > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
    warning @rails/webpacker > node-sass > request > har-validator@5.1.5: this library is no longer supported
    warning @rails/webpacker > webpack > watchpack > watchpack-chokidar2 > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
    warning @rails/webpacker > webpack > watchpack > watchpack-chokidar2 > chokidar > fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
    warning @rails/webpacker > webpack > micromatch > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
    warning @rails/webpacker > webpack > micromatch > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated`

In other words, I would love to deploy my Shopify Apps without all these webpacker warnings.

**Comment by yoonwaiyan** [comment]
These warnings are all related to webpacker gem, you may get better luck there. Apparently, these are all dependency issues which is going to be there from time to time due to the fast-changing nature of the JavaScript ecosystem. If you would prefer to silent the warnings, see this issue: https://github.com/yarnpkg/yarn/issues/6672 .

**Comment by andyw8** [comment]
Closing as this isn't due to `shopify_app`.
